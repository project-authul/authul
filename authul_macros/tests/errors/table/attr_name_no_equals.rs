#[authul_macros::authul_table(name)]
struct Foo;

fn main() {
}
