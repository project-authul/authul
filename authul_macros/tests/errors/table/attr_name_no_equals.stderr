error: premature end of macro arguments; expected '=' after 'name'
 --> tests/errors/table/attr_name_no_equals.rs:1:1
  |
1 | #[authul_macros::authul_table(name)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `authul_macros::authul_table` (in Nightly builds, run with -Z macro-backtrace for more info)
